app/pipelines/customer_chatbot_pipeline.py

```
@@ -0,0 +1,624 @@
+            )
+
+        # 분해 결과가 없는 턴은 본문이 비므로 메시지를 보내지 않는다(PRD 2.5).
+        if not extraction.guide_search_queries:
+            return ""
+
+        try:
+            response = self.guide_responder.respond(
+                guide_search_queries=extraction.guide_search_queries,
+                session=self.session,
+            )
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)